DQMOffline/Alignment/python/ALCARECOMuAlDQM_cff.py

```diff
@@ -233,4 +291,4 @@
 #)
 #ALCARECOMuAlZMuMuDQM = cms.Sequence( ALCARECOMuAlZMuMuTrackingDQM + ALCARECOMuAlZMuMuMuAlDQM + ALCARECOMuAlZMuMuHLTDQM)
 ALCARECOMuAlZMuMuDQM = cms.Sequence( ALCARECOMuAlZMuMuTrackingDQM + ALCARECOMuAlZMuMuMuAlDQM)
-
+ALCARECOMuAlZMuMuGeneralTracksDQM = cms.Sequence( ALCARECOMuAlZMuMuGeneralTracksTrackingDQM + ALCARECOMuAlZMuMuGeneralTracksDQM)
```

Comment:
@lpernie this line cannot be correct:
```
ALCARECOMuAlZMuMuGeneralTracksDQM = cms.Sequence( ALCARECOMuAlZMuMuGeneralTracksTrackingDQM 
+ 
ALCARECOMuAlZMuMuGeneralTracksDQM)
```
please rename the sequence to something else than the clone of `_MuonAl.muonAlignment.clone`